manager.py

```diff
@@ -35,16 +37,25 @@ def remove_worker(conn, host):
 
 # connect_to_master method is used to connect to master coordinator at the start-up.
 # Citus docker-compose has a dependency mapping as worker -> manager -> master.
-# This means that whenever manager is created, master is already there, so we should
-# always be able to successfully connect
+# This means that whenever manager is created, master is already there, but it may
+# not be ready to accept connections. We'll try until we can create a connection.
 def connect_to_master():
     citus_host    = environ.get('CITUS_HOST', 'master')
     postgres_pass = environ.get('POSTGRES_PASSWORD', '')
     postgres_user = environ.get('POSTGRES_USER', 'postgres')
     postgres_db   = environ.get('POSTGRES_DB', postgres_user)
 
-    conn = psycopg2.connect("dbname=%s user=%s host=%s password=%s" %
-                            (postgres_db, postgres_user, citus_host, postgres_pass))
+    conn = None
+    while conn is None:
+        try:
+            conn = psycopg2.connect("dbname=%s user=%s host=%s password=%s" %
+                                    (postgres_db, postgres_user, citus_host, postgres_pass))
+        except psycopg2.OperationalError as error:
+            print("Could not connect to %s, trying again in 1 second" % citus_host)
```

Comment:
I looked into psycopg2 docs to find a better way of connection polling but I couldn't. After testing, I will approve the pr